### PR TITLE
Feature/custom error messages boolean

### PR DESCRIPTION
**Pull request by aaronjameslang**
#2

### CODE REVIEW

**Comment by i-am-tom** [comment]
I am really enjoying your tests, hah! One _tiny_ request: can you make the variables `$error` instead of `$errorMessage`? I'm thinking that `string` will imply that it isn't a code. Also, would you mind updating the README? I suspect there are more PRs coming, so I won't say anything more than that, otherwise, this is wonderful!

**Comment by aaronjameslang** [comment]
Will do :smile:

File: README.md [hunk 1]
```diff
@@ -166,6 +166,10 @@ Schemer\Validator::boolean() // Accept only boolean values.
     ->true() // Accept only 'true'.
     ->false() // Accept only 'false'.
 
+Schemer\Validator::boolean("George says no") // Accept only boolean values, with a custom error.
```
Comment (i-am-tom) [inline README.md]: I'm worried about this making the README too bloated; maybe we'll go for a note at the top ("all these take a custom $error string" etc) and then just an `$error` param in all the functions already there? What do you think?

Comment (aaronjameslang) [reply README.md]: Sounds sensible

**Comment by aaronjameslang** [comment]
@i-am-tom How's that for the README?

**Comment by i-am-tom** [comment]
Yep, works for me! Thanks for this :D

**Comment by i-am-tom** [comment]
Turns out I have to merge it _before_ closing the PR... oops!

**Pull request merged**